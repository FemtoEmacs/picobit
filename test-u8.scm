(define x (u8vector 65 67 118 10))
;; (define x "ACD\n")
;; (display (string-append (substring x 0 2) "\n"))
;; (u8vector-set! x 2 70)
(display (u8vector-ref x 2))
;; (+ (substring x 0 0) (substring x 3 3))
