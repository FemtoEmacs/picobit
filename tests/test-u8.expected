:0080000080
:10800000FBD72B0580036114811C400080036110A5
:10801000812640008117400081124000810F4000FE
:10802000811140008003610D800401218042000223
:1080300080270122800A0123802701248024012532
:108040008071012780040128806E012980070002C9
:108050008027012A8077012B800C00028073012C7D
:1080600080240002800E0002806A00028073012DCD
:10807000800600028067012E8005011680400118ED
:10808000800B01198040011A8032011B8027011DDD
:108090008066011E8005011F806901208078000232
:1080A000807D0113806601158070010E8078010EBD
:1080B00002B402F35402B402ED5302B4029A500621
:1080C0000708406352070BB00268514206DE06C142
:1080D000B5E3EE42EBB5DEEEA105B5D9EEA106069D
:1080E000DEB5D2EEA106EBB5CCEEA105B5C7EE41EB
:1080F0000609DF4106DEB5BDEE41EBB5B8EEA105E0
:10810000B5B3EEA104A104B001FFB5A9EEA104A18D
:108110000CB001FFB59FEEA1040405064063B00159
:10812000FFB592EE04050640630405064063B00106
:10813000FFB582EED80120DAB8954321DBB6800284
:1081400021CC9A21CE2161EE2022CFB67200EF2001
:10815000B6800120DA9EA02607E620B558EEA026BC
:1081600007E600EF20C0B8BF20B582B64801200462
:10817000C8B8AEA03121B5A402B583CDDCEF02A012
:1081800034210EC5C122CD2102CDB4018E71012151
:108190000EC89220EF210E446221B662010421C272
:1081A000EF2002B559B75520CCB8B8A02C07E6206F
:1081B000CEB51FEE20CFB6800120D296A02D07E6C7
:1081C00000EF20CC9DA02407E620CEB505EE20CF01
:1081D000B666A109B00135EE20B00152EEA02D0720
:1081E000E600EF20D995A10AB1013520F09C209539
:1081F000A108B10135A107B10135A10BB1013502CB
:108200002022D39201EF20CCB8D921CC9F20CE22BE
:10821000CEB56C9620CF22CFB66500EF20E9B8BE70
:1082200021E9B8B82022B6800220EB222202CDCD6F
:10823000B4023471012022EBC7B89F22222205C26A
:10824000B680032204C79201EF2023DE2224DEC77A
:108250009720222405C2B66A00EF00EF00EF00B8B5
:108260007BB74100B836B7250220DD2105C223B611
:1082700080032104B58E9B202224DF202205C22406
:10828000B66F20EF022022CA222202CDCDB4029185
:108290007101209220EF2222C7EFFF20B68001203B
:1082A000B5B92102CDB402A9710120DD2202CDB4FD
:1082B00002B37101212102CDCDB402BD70002104B1
:1082C00022B583EE21EF03202224CEDF22CFD2D4A9
:1082D00098202205C124CFB66D00EF012004B6809E
:1082E0000220CC9720CF2205C1B67521EF012007CF
:1082F000E600EF022004C8212302CDCDB4030071B3
:10830000012204C822242302CDCDCDB4030F710174
:1083100023B8DC21B0019C23222602CDCDCDB403AD
:1083200022710122B8C621B0019C21242602CDCDA4
:10833000CDB4033571012022242602CDCDCDCDB49C
:10834000034370002121C4232502CDCDB403507115
:108350000122B893219220EF2298019420B1019C30
:1083600020EF21B87BB77500B86EB76A21B73B2103
:02837000B7252F
:00000001FF
